src/odemis/acq/milling/test/openfibsem_millmng_test.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Feb 2025
+
+Copyright © Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import glob
+import logging
+import os
+import unittest
+import time
+import odemis
+from odemis import model
+from odemis.acq.milling.tasks import load_milling_tasks, __file__ as MILLING_PATH
+from odemis.acq.milling.openfibsem import run_milling_tasks_openfibsem, OpenFIBSEMMillingTaskManager
+
+from odemis.util import testing
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+METEOR_FISBEM_CONFIG = CONFIG_PATH + "sim/meteor-fibsem-sim.odm.yaml"
+METEOR_FISBEM_CONFIG = "/home/patrick/development/odemis/install/linux/usr/share/odemis/sim/meteor-fibsem-sim.odm.yaml"
+MILLING_TASKS_PATH = os.path.join(os.path.dirname(MILLING_PATH),  "milling_tasks.yaml")
+
+class TestOpenFIBSEMMillingManager(unittest.TestCase):
+
+    """
+    Test the OpenFIBSEM Milling Manager
+    Requires the xt simulator to be running
+    """
+    MIC_CONFIG = METEOR_FISBEM_CONFIG
+
+
+    @classmethod
+    def setUpClass(cls):
+        testing.start_backend(cls.MIC_CONFIG)
+        cls.microscope = model.getMicroscope()
+        cls.milling_tasks = load_milling_tasks(MILLING_TASKS_PATH)
+
+    def test_estimate_total_milling_time(self):
+        """Test the estimate_total_milling_time function"""
+        openfibsem_milling_manager = OpenFIBSEMMillingTaskManager(None,
+                                                                  self.milling_tasks)
```

Comment:
`self.milling_tasks` is a dict, while `OpenFIBSEMMillingTaskManager` expects a list

Reply:
internally it checks for dicts and converts to the list of tasks, i've updated the signature to show it takes dict